### PR TITLE
deployment manager uses new stackdriver agents

### LINKED ISSUE #806: deployment manager should disable legacy Stackdriver agents
- For GKE 1.10 and above
- And then bootstrapper can deploy the new agents.

https://cloud.google.com/monitoring/kubernetes-engine/

### PR DESCRIPTION
**Pull request by lluunn**
Fix #806  #807 

https://cloud.google.com/deployment-manager/docs/configuration/supported-resource-types

/cc @jlewi @kunmingg

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/814)

### CODE REVIEW

File: docs/gke/configs/cluster.jinja [hunk 1]
```diff
@@ -36,6 +36,11 @@ resources:
         - https://www.googleapis.com/auth/devstorage.read_only
         - https://www.googleapis.com/auth/logging.write
         - https://www.googleapis.com/auth/monitoring
+      # Logging and monitoring have default value [logging/monitoring].googleapis.com
+      # if not set. We set it to none and will deploy the new Stackdricer Kubernetes.
```
Comment (jlewi) [inline docs/gke/configs/cluster.jinja]: Stackdriver Kubernetes agents

**Comment by lluunn** [comment]
/hold
We might not need this.
We can disable Stackdriver agent after cluster created, so can be done by bootstrapper.
https://cloud.google.com/kubernetes-engine/docs/how-to/logging#disabling_logging

**Comment by lluunn** [comment]
/hold cancel
@jlewi How about we just deploy the new agents using deployment manager?
I think it's the better way, comparing to disabling old agents here, and deploy new ones at bootstrapper.
Also it's much easier.

File: docs/gke/configs/cluster.jinja [hunk 2]
```diff
@@ -28,14 +28,21 @@ resources:
       name: {{ CLUSTER_NAME }}
       # 
       initialNodeCount: {{ properties['initialNodeCount'] }}
-      # We need 1.9 to support GPUs without requiring Alpha.
-      initialClusterVersion: 1.9.6-gke.1
+      # TODO: remove alpha when 10.2 is public.
```
Comment (jlewi) [inline docs/gke/configs/cluster.jinja]: Can you file an issue an reference the issue here? Could we add an option that can be enabled via the YAMl spec to enable Alpha features or not? And then selectively enable the logging and monitoring service below?

**Comment by jlewi** [comment]
Using deployment manager is better to running gcloud commands.

**Comment by lluunn** [comment]
@jlewi comment addressed, PTAL

**Pull request merged**